app/helpers/products_helper.rb

```diff
@@ -41,9 +41,11 @@ def public_calendar_link(product)
   def product_url_hint(product)
     product_url_name = product.try(:url_name).presence || "url-name"
 
-    url_name_hint = send("facility_#{product.class.name.downcase}_url",
-                         current_facility.url_name,
-                         product_url_name)
+    send(
+      "facility_#{product.class.name.underscore}_url",
```

Comment:
Change from `downcase` to `underscore` is the result of my spike being on `TestProduct`, which should be `test_product` instead of `testproducts`.